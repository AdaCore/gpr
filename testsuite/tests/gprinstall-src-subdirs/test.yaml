description:
    check gprinstall --src-subdirs
driver: python_script
