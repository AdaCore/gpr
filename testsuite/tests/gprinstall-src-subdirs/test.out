OK counters.adb
OK instrumented counters.adb
OK counters.sid
OK instrumented counters.sid
Reaching end of test :  1

Reaching end of test :  9

